# Publish the detected circles poses with their respective labels
string label
geometry_msgs/PoseStamped pose